### PR TITLE
[#18] [UI] As a user, I can see the survey detail page

### LINKED ISSUE #18: [UI] As a user, I can see the survey detail page
## Why
When choosing a survey, it navigates the user to the survey detail page.

## Acceptance Criteria
- Create the Survey Detail page with the below components:
  - The survey image.
  - The survey's title.
  - The survey's description.
  - The Start Survey button.
  - The background.
[Figma](https://www.figma.com/file/i1AI0IzCWuJApX5CK0DmiW/Challenge---Web-App?type=design&node-id=70-331&t=apcSQ1nCik2OLC0u-0)
image

### PR DESCRIPTION
**Pull request by manh-t**
Close #18 

## What happened 👀

- Implement the UI for the [Survey detail screen](https://www.figma.com/file/i1AI0IzCWuJApX5CK0DmiW/Challenge---Web-App?node-id=70%3A331&mode=dev).

## Insight 📝

N/A

## Proof Of Work 📹

image

### CODE REVIEW

**Comment by manh-t** [comment]
🚀 Deployed on https://64edb40bb8fdeb3abc20a052--manh-react-survey.netlify.app

**Comment by manh-t** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Messages</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:book:</td>
      <td>Congrats, coverage is good</td>
    </tr>
  </tbody>
</table>

## Coverage in All Files
File | Line Coverage | Statement Coverage | Function Coverage | Branch Coverage
---- | ------------: | -----------------: | ----------------: | --------------:
[src/tests/error.ts](../blob/cecb690/src/tests/error.ts) | (2/3) 67% | (2/3) 67% | (1/2) 50% | (1/3) 33%
[src/../Content/index.tsx](../blob/cecb690/src/components/Dashboard/Content/index.tsx) | (14/15) 93% | (14/15) 93% | (7/7) 100% | (6/9) 67%
[src/../v1/requestManager.ts](../blob/cecb690/src/lib/request/v1/requestManager.ts) | (19/20) 95% | (19/20) 95% | (6/6) 100% | (3/5) 60%
[src/App.tsx](../blob/cecb690/src/App.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (0/0) NaN%
[src/index.tsx](../blob/cecb690/src/index.tsx) | (3/3) 100% | (3/3) 100% | (0/0) NaN% | (0/0) NaN%
[src/../Authentication/index.ts](../blob/cecb690/src/adapters/Authentication/index.ts) | (4/4) 100% | (4/4) 100% | (2/2) 100% | (0/0) NaN%
[src/adapters/Base/index.ts](../blob/cecb690/src/adapters/Base/index.ts) | (8/8) 100% | (8/8) 100% | (2/2) 100% | (2/2) 100%
[src/adapters/BaseAuth/index.ts](../blob/cecb690/src/adapters/BaseAuth/index.ts) | (10/10) 100% | (10/10) 100% | (4/4) 100% | (2/2) 100%
[src/adapters/Survey/index.ts](../blob/cecb690/src/adapters/Survey/index.ts) | (1/1) 100% | (1/1) 100% | (1/1) 100% | (0/0) NaN%
[src/adapters/User/index.ts](../blob/cecb690/src/adapters/User/index.ts) | (1/1) 100% | (1/1) 100% | (1/1) 100% | (0/0) NaN%
[src/components/Alert/index.tsx](../blob/cecb690/src/components/Alert/index.tsx) | (3/3) 100% | (3/3) 100% | (2/2) 100% | (0/0) NaN%
[src/../Empty/index.tsx](../blob/cecb690/src/components/Dashboard/Empty/index.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (0/0) NaN%
[src/../Header/index.tsx](../blob/cecb690/src/components/Dashboard/Header/index.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (9/9) 100%
[src/../index.tsx](../blob/cecb690/src/components/ElevatedButton/index.tsx) | (3/3) 100% | (3/3) 100% | (1/1) 100% | (0/0) NaN%
[src/../LoadingDialog/index.tsx](../blob/cecb690/src/components/LoadingDialog/index.tsx) | (2/2) 100% | (2/2) 100% | (1/1) 100% | (0/0) NaN%
Other (29 more) | (195/195) 100% | (195/195) 100% | (71/71) 100% | (54/58) 93%
Total | (274/277) 99% | (274/277) 99% | (102/103) 99% | (77/88) 88%

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against a933c5784a02b139b080c418d51a099c197fcd1f
</p>

File: src/screens/Survey/index.test.tsx [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  const TestComponent = (): JSX.Element => {
+    return (
+      <TestWrapper>
+        <SurveyScreen />
+      </TestWrapper>
+    );
+  };
+
+  it('renders Survey screen and its components', () => {
+    render(<TestComponent />);
```
Comment (tyrro) [inline src/screens/Survey/index.test.tsx:20]:
```suggestion
    render(<TestComponent />);
    
```

**Review by tyrro — APPROVED** [review]

File: src/screens/Survey/index.tsx [hunk 2]
```diff
@@ -1,10 +1,49 @@
...
+  coverImage: 'survey__cover-image',
+  title: 'survey__title-text',
+  description: 'survey__description-text',
+  startSurveyButton: 'survey__start-survey-button',
+};
 
-  return <div>Dashboard detail {id}</div>;
+const SurveyScreen = (): JSX.Element => {
+  return (
+    <BackgroundImage backgroundUrl="https://dhdbhh0jsld0o.cloudfront.net/m/1ea51560991bcb7d00d0_">
```
Comment (nvminhtue) [inline src/screens/Survey/index.tsx]: It's better to update the component's name, `BackgroundImage` sounds like a `img` tag, not a component has children

File: src/assets/images/icons/arrow-back.svg [hunk 3]
```diff
@@ -0,0 +1,3 @@
+<svg width="28" height="30" viewBox="0 0 28 30" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M16.5371 25.0827L7.29215 15.9458C6.90262 15.5612 6.90262 14.9398 7.29215 14.5542L16.5371 5.41733C17.0995 4.86089 18.0144 4.86089 18.5777 5.41733C19.14 5.97378 19.14 6.87687 18.5777 7.43332L10.6691 15.2505L18.5777 23.0657C19.14 23.6231 19.14 24.5262 18.5777 25.0827C18.0144 25.6391 17.0995 25.6391 16.5371 25.0827Z" fill="white"/>
```
Comment (nvminhtue) [inline src/assets/images/icons/arrow-back.svg]:
```suggestion
<path fill-rule="evenodd" clip-rule="evenodd" d="M16.5371 25.0827L7.29215 15.9458C6.90262 15.5612 6.90262 14.9398 7.29215 14.5542L16.5371 5.41733C17.0995 4.86089 18.0144 4.86089 18.5777 5.41733C19.14 5.97378 19.14 6.87687 18.5777 7.43332L10.6691 15.2505L18.5777 23.0657C19.14 23.6231 19.14 24.5262 18.5777 25.0827C18.0144 25.6391 17.0995 25.6391 16.5371 25.0827Z" fill="currentColor"/>
```

File: src/screens/Survey/index.tsx (hunk 2 above)
Comment (manh-t) [reply src/screens/Survey/index.tsx]: @nvminhtue I have a name in mind, how about calling this widget `MainView` since it's always a parent widget? 😃

**Review by nvminhtue — APPROVED** [review]

**Pull request merged**